LEGO1/mxomni.cpp

```diff
@@ -42,10 +42,16 @@ void MxOmni::Init()
   m_unk64 = NULL;
 }
 
-// OFFSET: LEGO1 0x100b0090 STUB
-void MxOmni::vtable0x20()
+// OFFSET: LEGO1 0x100b0090
+long MxOmni::Start(MxDSAction* ds)
```

Comment:
The return type should probably be `MxResult`. Parameters should be prefixed with `p_`. We should also pass a reference instead of a pointer here, so I'd suggest `MxDSAction &p_dsAction` as a signature

Reply:
If I use a reference instead of a pointer, it wouldn't match anymore because MxOmni::Start and MxStreammer::Unknown100b99b0 checks if it's null.